### ISSUE TITLE
How to split train,test and validatation data?

### ISSUE THREAD

**Issue by Vaibhav-ML** [fizyr/keras-retinanet#1504]
Hi,

    I have a custom dataset which is running smoothly. I want to now split it into train, test and validation set.

How do I do this in this project?

**Comment by hlmhlr** [comment]
Dear @Vaibhav-ML, you are required to split the data into train, test and val sets before starting the training in this project. In which format you want to train (cvs, pascalvoc, coco, etc), normally csv for custom data, so choose the format and make the dataset directories accordingly. You may either create your own code to convert the original format or may search over the internet. Finally, provide the path of dataset directory for the training.
